docs/docs/fallback-handoff.mdx

```diff
@@ -283,6 +285,57 @@ rules:
   - active_loop: action_two_stage_fallback
 ```
 
+### 4. Defining an ultimate fallback action
+
+To define the bot's response when the user denies the rephrased intent, define the response `utter_default`:
+
+```yaml-rasa title="domain.yml"
+responses:
+  utter_default:
+  - text: I'm sorry, I can't help you.
+```
+
+Or, you can customize `action_default_fallback` for more complex behavior by writing a [Custom Action](./actions.mdx#custom-actions). 
+For example, if you want the bot to call a human and stop interacting with the user:
+
+```python title="actions.py"
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.events import UserUtteranceReverted
+from rasa_sdk.executor import CollectingDispatcher
+
+class ActionDefaultFallback(Action):
+    def name(self) -> Text:
+        return "action_default_fallback"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        # tell the user they are being passed to a customer service agent
+        dispatcher.utter_message(text="I am passing you to a human...")
+        
+        # assume there's a function to call customer service
+        # pass the tracker so that the agent has a record of the converstation between the user
+        # and the bot for context
+        call_customer_service(tracker)
+     
+        # pause the tracker so that the bot stops responding to user input
+        return [ConversationPaused(), UserUtteranceReverted()]
+```
+
+  :::caution Events Returned By A Custom Ultimate Fallback Action
+    It is a good practice to include `UserUtteranceReverted()` as one of the events returned by your custom
+    `action_default_fallback`. Not including this event will cause the tracker to include all events that happened
+    during the Two-Stage Fallback process which could intefere with subsequent action predictions from the bot's policy
+    pipeline. It is better to treat events that occured during the Two-Stage Fallback process as if they did not happen
+    so that your bot can apply its rules or memorised stories to correctly predict the next action. 
+  :::
```

Comment:
@melindaloubser1 regarding your comment on on `UserUtteranceReverted()`, what do you think of this update?

Reply:
That sounds good to me
```suggestion
  :::caution Events Returned By A Custom Ultimate Fallback Action
    You should include `UserUtteranceReverted()` as one of the events returned by your custom
    `action_default_fallback`. Not including this event will cause the tracker to include all events that happened
    during the Two-Stage Fallback process which could intefere with subsequent action predictions from the bot's policy
    pipeline. It is better to treat events that occured during the Two-Stage Fallback process as if they did not happen
    so that your bot can apply its rules or memorised stories to correctly predict the next action. 
  :::
```

Sounds good! Just one language update since we have a "kill list" for writing docs  =)